### ISSUE TITLE
vkBasalt indicator is misleading

### ISSUE THREAD

**Issue by itspngu** [flightlessmango/MangoHud#651]
vkBasalt supports toggling its effects on and off with a keypress, but MangoHud always shows "vkBasalt: on", regardless of effect state. One could argue that vkBasalt is "on" as soon as its lib is loaded - which is how the [current indicator logic](https://github.com/flightlessmango/MangoHud/blob/e7de7b5b933c20a7ae88d445678390a8d4bdf7ac/src/overlay.cpp#L771) queries vkBasalt state - but it'd be nice if it accurately reflected wether vkBasalt's *effects* are on or off.

I'm surprised this hasn't been raised as an issue yet, because I just spent about half an hour trying to figure out why the vkBasalt toggle doesn't work, when infact it was working (FXAA turning on and off once I actually took a good look at the output) and I've just been staring at "vkBasalt: on" like an idiot the whole time. Maybe I need glasses, dunno.

**Comment by flightlessmango** [comment]
I don't think this is possible, we don't have access to internals of another layer

**Comment by flightlessmango** [comment]
vkbasalt is not being developer anymore, it's unlikely this can be resolved
